sklearn/datasets/base.py

```diff
@@ -242,6 +242,122 @@ def load_files(container_path, description=None, categories=None,
                  DESCR=description)
 
 
+def load_data(module_path, data_file_name):
+    """
```

Comment:
I mean PEP257 recommends a one-line summary on the same line as `"""`, followed optionally by blank line and further details.